docs/src/cli/manage-stake-accounts.md

```diff
@@ -0,0 +1,74 @@
+# Manage Stake Accounts
+
+If you want to delegate stake to many different validators, you will need
+to create a separate stake account for each. If you follow the convention
+of creating the first stake account at seed "0", the second at "1", the
+third at "2", and so on, then the `solana-stake-accounts` tool will allow
```

Comment:
If this tool can be used to create derived accounts, do we need to expose the "0", "1" seed string logic?

Reply:
Lots of features can be added.  In this case, it'd be an `--index` option to the `new` command.  All the other commands operate on *all* derived accounts.  The tool isn't meant to redirect existing commands to a derived account.

Reply:
Okay, added `solana-stake-accounts new --index <NUMBER>`